lib/config/fileconf.go

```diff
@@ -2207,10 +2207,17 @@ type WindowsDesktopService struct {
 	// Hosts is a list of static, AD-connected Windows hosts. This gives users
 	// a way to specify AD-connected hosts that won't be found by the filters
 	// specified in `discovery` (or if `discovery` is omitted).
+	//
+	// Deprecated: prefer StaticHosts instead.
 	Hosts []string `yaml:"hosts,omitempty"`
```

Comment:
Should we rename this to `ADHosts`?

(Keep the YAML as `hosts` so as not to break configurations)